### PR TITLE
[properties] physical -> logical

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

-----

⚠️ Don't merge before UI Diff is available again

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3312/storybook

File: packages/scss/src/commons/core.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@ $overflow: 'hidden', 'auto', 'visible', 'scroll';
 /* Tokens */
 
 @mixin spacing($boxModel, $boxDirection, $key, $value, $suffix: '!important') {
-	.pr-u-#{transform.camelize($boxModel)}#{transform.capitalize($boxDirection)}#{transform.capitalize($key)} {
+	.pr-u-#{transform.camelize($boxModel)}#{transform.capitalize(transform.camelize($boxDirection))}#{transform.capitalize($key)} {
```
Comment (vvalentin-lucca) [inline packages/scss/src/commons/core.scss:38]: please review here

File: packages/scss/src/commons/core.scss [hunk 2]
```diff
@@ -8,8 +8,8 @@
 
 $contents: '100\\%', 'fit-content';
 $boxModel: 'margin', 'padding', 'border';
-$boxDirection: '', 'top', 'bottom', 'left', 'right', 'inline', 'block';
-$corners: '', 'top-left-', 'top-right-', 'bottom-left-', 'bottom-right-';
+$boxDirection: '', 'top', 'bottom', 'left', 'right', 'inline', 'block', 'block-start', 'block-end', 'inline-start', 'inline-end'; // top, bottom, left and right are deprecated
```
Comment (vvalentin-lucca) [inline packages/scss/src/commons/core.scss:11]: please review here

File: packages/ng/schematics/tokens-spacing/tests/template.output.html [hunk 3]
```diff
@@ -1,20 +1,36 @@
-<div class="timeline mod-number pr-u-paddingLeft600 pr-u-paddingRight600 pr-u-paddingTop200 pr-u-paddingBottom400"></div>
+<div
+	class="timeline mod-number pr-u-paddingInlineStart600 pr-u-paddingInlineEnd600 pr-u-paddingBlockStart200 pr-u-paddingBlockEnd400"
```
Comment (vvalentin-lucca) [inline packages/ng/schematics/tokens-spacing/tests/template.output.html:2]: please review here

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3312/storybook

**Pull request closed without merging**